inference/worker/__main__.py

```diff
@@ -23,19 +24,37 @@ def main():
     signal.signal(signal.SIGINT, terminate_worker)
     logger.info(f"Inference protocol version: {inference.INFERENCE_PROTOCOL_VERSION}")
 
-    if settings.model_id != "_lorem":
-        if "llama" in settings.model_id:
-            tokenizer: transformers.PreTrainedTokenizer = transformers.LlamaTokenizer.from_pretrained(settings.model_id)
+    model_config = model_configs.MODEL_CONFIGS.get(settings.model_config_name)
+    if model_config is None:
+        logger.error(f"Unknown model config name: {settings.model_config_name}")
+        sys.exit(2)
+
+    if not model_config.is_lorem:
+        if model_config.is_llama:
+            tokenizer: transformers.PreTrainedTokenizer = transformers.LlamaTokenizer.from_pretrained(
```

Comment:
is this special handling still necessary with latest transformers version? (e.g. maybe wee need to update our json config files?)

Reply:
with the latest verison, probably none, but the version I use right now is a very unholy merge of two forks...

Reply:
very likely we can remove this in the future once the llama models have arrived at the version used in text-generation-inference